seeding/create-user.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# If you're on Windows, run bash create-user.sh -w
+# Otherwise, run bash create-user.sh
+
+# Import common functions
+source ./seed.sh
+
+# Run SQL script in docker container
+run_sql_script "
+INSERT INTO users (
+  first_name,
+  last_name,
+  auth_id,
+  role
+)
+SELECT 
+  '$FIRST_NAME',
+  '$LAST_NAME',
+  '$AUTH_ID',
+  '$ROLE'
+WHERE NOT EXISTS (
+  SELECT 1
+  FROM users
+  WHERE first_name = '$FIRST_NAME'
+    AND last_name = '$LAST_NAME'
+);
+"
```

Comment:
If I'm correct here, this inserts the user you want to add only if a user with the same first and last name doesn't exist? I'm probably missing the point of this, but multiple users can have the same name no?

Reply:
this is just a check to make sure you're not inserting yourself into the db if you already exist